src/lib/helpers.js

```diff
@@ -1,5 +1,10 @@
 export function isInitialOrHasNoParents(chunk) {
-  return chunk.isInitial() || chunk.getParents().length === 0;
+  const parentCount = Array.from(
+    chunk.groupsIterable,
+    (chunkGroup) => chunkGroup.getParents().length
+  ).reduce((current, sum) => current + sum);
```

Comment:
This was giving me an error:
```
TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>:null:null)
    at isInitialOrHasNoParents (/home/sciyoshi/workspace/web/webpack/extract-text-webpack-plugin/dist/lib/helpers.js:15:102)
```
Changing the line to `).reduce((current, sum) => current + sum, 0);` fixed it.

Reply:
Thank you! I noticed that just yesterday, along with failing integration test. I didn’t push yet because I couldn’t make it pass everything.

I’ll fix and push the obvious low hanging fruits I missed and will keep working on the tests.

Expect an update ~12 hours later.

Reply:
@horo42 also can you add tests around this?